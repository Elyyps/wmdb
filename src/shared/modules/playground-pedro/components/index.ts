export * from "./playground-pedro.component";
