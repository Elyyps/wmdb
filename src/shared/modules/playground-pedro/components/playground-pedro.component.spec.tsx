import React from "react";
import { shallow } from "enzyme";
import { PlaygroundPedroComponent } from "./playground-pedro.component";

describe("[PlaygroundPedroComponent]", () => {
  it("should render without crashing", () => {
    const renderedComponent = shallow(<PlaygroundPedroComponent />);

    expect(renderedComponent).toMatchSnapshot();
  });
});
