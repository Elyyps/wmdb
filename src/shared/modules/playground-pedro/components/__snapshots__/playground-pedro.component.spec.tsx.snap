// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[PlaygroundPedroComponent] should render without crashing 1`] = `
<div
  className="main"
>
  <HeroComponent
    heroModule={
      Object {
        "buttonTitle": "Toon 500 uijtes",
        "dropdownCompanyItems": Array [
          Object {
            "text": "Item #1",
            "value": 1,
          },
          Object {
            "text": "Item #2",
            "value": 2,
          },
          Object {
            "text": "Item #3",
            "value": 3,
          },
        ],
        "dropdownCompanyPlaceholder": "Kies een bedrijfsuitje categorie",
        "image": "img01.png",
        "keywordPlaceholder": "Zoekwoord",
        "numberPersonsPlaceholder": "Aantal personen",
        "provincePlaceholder": "Provincie of Plaats",
        "title": "Wat gaan we dít jaar doen?! Vergelijk online gemakkelijk álle bedrijfsuitjes van Nederland",
      }
    }
    onSubmit={[Function]}
  />
  <CategoryOverviewComponent
    categoryOverviewModule={
      Object {
        "backgroundColor": "#e8f2fc",
        "items": Array [
          Object {
            "count": 200,
            "icon": "spel.svg",
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "count": 200,
            "icon": "spel.svg",
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "count": 200,
            "icon": "spel.svg",
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "count": 200,
            "icon": "spel.svg",
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "count": 200,
            "icon": "spel.svg",
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "count": 20,
            "icon": "spel.svg",
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
          Object {
            "image": "img02.png",
            "link": "/",
            "title": "Musea",
          },
        ],
        "paddingBottom": "72px",
        "paddingTop": "72px",
        "title": "Kies direct een categorie om het aanbod te bekijken!",
      }
    }
  />
  <div
    className="uk-container"
    style={
      Object {
        "maxWidth": "500px",
      }
    }
  >
    <ArrowPanelListComponent
      icon="eten.svg"
      items={
        Array [
          Object {
            "link": "/",
            "title": "Wat is Lorem Ipsum?",
          },
          Object {
            "link": "/",
            "title": "Hoe lang zijn mijn aanbiedingen of vacatures zichtbaar op indebuurt?",
          },
          Object {
            "link": "/",
            "title": "Hoe kan ik mijn (wekelijkse) aanbieding of vacature aanleveren?",
          },
          Object {
            "link": "/",
            "title": "Hoe lang staat mijn artikel op de homepagina, waar komt het vandaan?",
          },
          Object {
            "link": "/",
            "title": "Waar kan ik het vinden?",
          },
          Object {
            "link": "/",
            "title": "Waar kan ik het vinden?",
          },
        ]
      }
      linkMore={
        Object {
          "link": "/",
          "title": "Alle veelgestelde vragen over musea",
        }
      }
      title="Spel & Ontspanning"
    />
    <BodyTextComponent
      html="
<div style=\\"padding-bottom: 64px\\">
<h1 style=\\"margin-top: 64px\\">I'm the BodyText Module</h1>
<h2>I came from a plein HTML string</h2>
<span style=\\" backgroundColor: red; min-width: 100%; width: 100%; color: white; text-align: center;\\">And I support inline style</span> 
</div>
"
    />
    <Checkbox
      label="Checkbox Label"
    />
    <Checkbox
      isChecked={true}
      label="Checkbox checked"
    />
    <div
      className="components"
      style={
        Object {
          "display": "flex",
          "flexDirection": "column",
        }
      }
    >
      <Button
        title="Button primary"
      />
      <Button
        title="Button secondary"
        variant="secondary"
      />
    </div>
  </div>
</div>
`;
